src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -44,11 +55,45 @@ private void handleCommandEntered() {
         try {
             commandExecutor.execute(commandText);
             commandTextField.setText("");
+            commandHistory.add(commandText);
```

Comment:
I think this would be better called before executing the command; as the user might want to get the invalid command so that they can change it to a valid one